pages/contact.html

```diff
@@ -26,20 +27,15 @@
                 <div class="ui padded basic segment">
                     <h1 class="ui block header center blue aligned">Contact Details</h1>
                     <div class="ui center aligned basic segment">
-                        <p>Email School:<a href="mailto:info@mountsioncbssecondary.ie" target="_top">info@mountsioncbssecondary.ie</a></p>
-                        <p>Web: mountsioncbssecondary.ie</p>
-                        <p>Telephone Number: 051 377378</p>
-                        <p>Fax Number: 051-376468</p>
-                        <p>School Roll No. 64930i</p>
-                        <p>Address:</p>
-                        <p>Mount Sion CBS Secondary School,</p>
+                        <p><a href="mailto:info@mountsioncbssecondary.ie"><i class="envelope link icon"></a></i>info@mountsioncbssecondary.ie</p> 
+                        <p><a target="_blank" href="http://www.mountsioncbssecondary.ie"><i class="globe icon"></i></a>mountsioncbssecondary.ie</p>
+                        <p><a href="tel:+35351377378"><i class="phone icon"></i></a>051 377378</p>
+                        <p><a href="tel:+35351376468"><i class="fax icon"></a></i>051 376468</p>
+                        <p><i class="building icon"></i>School Roll Number: 64930I</p>
+                        <p><a target="_blank" href="https://www.google.com/maps?ll=52.256315,-7.116627&z=14&t=m&hl=en-US&gl=IE&mapclient=embed&cid=17885305901261910436"><i class="address card icon"></a></i>Mount Sion CBS Secondary School,</p>
```

Comment:
When I viewed this I didn't immediately know that the address below it was part of it. Perhaps you could add a line above it to show that the three lines here are part of the address.

Reply:
Yes, I will do that now.